bandit/formatters/html.py

```diff
@@ -263,6 +263,7 @@ def report(manager, fileobj, sev_level, conf_level, lines=-1):
     <b>Severity: </b>{severity}<br>
     <b>Confidence: </b>{confidence}<br>
     <b>File: </b><a href="{path}" target="_blank">{path}</a> <br>
+    <b>Line number: </b>{line_number}</a> <br>
```

Comment:
```suggestion
    <b>Line number: </b>{line_number}<br>
```

Reply:
Ahh didn't realize I could commit that suggestion from here. All set